src/com/company/Main.java

```diff
@@ -8,82 +8,60 @@
 
 
 public class Main {
-    public static void main(String[] args){
-
-        String txtfile = args[0];
-        File file = new File(txtfile);
-        checkforsupport(file);
-        checkforexistance(file);
+    final static String extention=".txt";
+    final static String specialChars="[^a-zA-Z0-9@]";
```

Comment:
constants should be defined in capitals and should be meaningful. specialChars does not serve the purpose of the regex used.

Reply:
done with the changes